### ISSUE TITLE
2.0rc1中model.prepare方法在何处有讲解？文档中似乎没有

### ISSUE THREAD

**Issue by A-Pai** [PaddlePaddle/Paddle#30668]
2.0rc1中model.prepare方法在何处有讲解？文档中似乎没有
参见文档https://www.paddlepaddle.org.cn/documentation/docs/zh/2.0-rc1/guides/02_paddle2.0_develop/01_quick_start_cn.html
中的“3.3 模型训练”，其中：
model.prepare(paddle.optimizer.Adam(parameters=mnist.parameters()),
              paddle.nn.CrossEntropyLoss(),
              paddle.metric.Accuracy())

“paddle.nn.CrossEntropyLoss()”与“paddle.metric.Accuracy()”的关系是？

**Comment by vslyu** [comment]
> 2.0rc1中model.prepare方法在何处有讲解？文档中似乎没有
> 参见文档https://www.paddlepaddle.org.cn/documentation/docs/zh/2.0-rc1/guides/02_paddle2.0_develop/01_quick_start_cn.html
> 中的“3.3 模型训练”，其中：
> model.prepare(paddle.optimizer.Adam(parameters=mnist.parameters()),
> paddle.nn.CrossEntropyLoss(),
> paddle.metric.Accuracy())
> 
> “paddle.nn.CrossEntropyLoss()”与“paddle.metric.Accuracy()”的关系是？

model.prepare方法见此：
https://www.paddlepaddle.org.cn/documentation/docs/zh/2.0-rc1/api/paddle/hapi/model/Model_cn.html#prepare-optimizer-none-loss-function-none-metrics-none

CrossEntropyLoss是交叉熵损失函数，模型根据Loss更新参数；Accuracy是根据模型输出的准确度，是评价模型好坏的指标；两者并没有关系

**Comment by A-Pai** [comment]
感谢耐心指导
